Fixing bug in PhaseIITreeMaker.cpp

ckaragian opened the pull request:
NDigitsPMTs and NDigitsLAPPDs are initialized to -9999. Then, they are incremented in order to count the number of PMT/LAPPD digits in the event. But this way, there is a -9999 offset to their values. So I initialize them to 0 before incrementing. Also I removed the totalPMTs and totalLAPPDs variables because they are basically not used.

## Describe your changes

## Checklist before submitting your PR
- [ ] This PR implements a single change (one new/modified Tool, or a set of changes to implement one new/modified feature)
- [ ] This PR alters the minimum number of files to affect this change
- [ ] If this PR includes a new Tool, a README and minimal demonstration ToolChain is provided
- [ ] If a new Tool/ToolChain requires model or configuration files, their paths are not hard-coded, and means of generating those files is described in the readme, with examples provided on /pnfs/annie/persistent
- [ ] For every `new` usage, there is a reason the data must be on the heap
- [ ] For every `new` there is a `delete`, unless I explicitly know why (e.g. ROOT or a BoostStore takes ownership)

## Additional Material
Attach any validation or demonstration files here. You may also link to relavant docdb articles.

ckaragian committed:
Fixing bug in PhaseIITreeMaker.cpp

NDigitsPMTs and NDigitsLAPPDs are initialized to -9999. Then, they are incremented in order to count the number of PMT/LAPPD digits in the event. But this way, there is a -9999 offset to their values. So I initialize them to 0 before incrementing. Also I removed the totalPMTs and totalLAPPDs variables because they are basically not used.

ckaragian committed:
Fixing typo

jminock reviewed (changes requested):
Hey Valantis, this looks good. Would you mind setting these two variables to 0 in the ResetVariableBranches function (should be lines 992 and 993) instead of where you did? In case these variables would be used elsewhere, making the change where I suggested will account for it. You are absolutely correct on deleting the unnecessary variables. Will be ready to merge if you can make the change!

ckaragian committed:
Update PhaseIITreeMaker.cpp

Initialize NDigitsPMTs and NDigitsLAPPDs variables in ResetVariables() function.

jminock reviewed (approved):
Thank you!! Looks good!
